common/src/main/java/tachyon/heartbeat/HeartbeatContext.java

```diff
@@ -46,6 +47,7 @@
     sTimerClasses.put(WORKER_BLOCK_SYNC, SLEEPING_TIMER_CLASS);
     sTimerClasses.put(WORKER_LINEAGE_SYNC, SLEEPING_TIMER_CLASS);
     sTimerClasses.put(WORKER_CLIENT, SLEEPING_TIMER_CLASS);
+    sTimerClasses.put(PIN_LIST_SYNC, SLEEPING_TIMER_CLASS);
```

Comment:
Rename to `WORKER_PIN_LIST_SYNC` to be consistent. Also please alphabetize, we're so close!